move_group/src/default_capabilities/move_action_capability.cpp

```diff
@@ -61,8 +61,26 @@ void move_group::MoveGroupMoveAction::initialize()
 void move_group::MoveGroupMoveAction::executeMoveCallback(const moveit_msgs::MoveGroupGoalConstPtr& goal)
 {
   setMoveState(PLANNING);
-  context_->planning_scene_monitor_->updateFrameTransforms();
 
+  const ros::Time now = ros::Time::now();
+  for (int i = 0; ; ++i)
+  {
+    ros::Duration age = now - context_->planning_scene_monitor_->getLastRobotStateUpdateTime();
+    if (age < ros::Duration(0))
+      break;
+
+    if (i >= 200)
```

Comment:
@davetcoleman yes, sure. thx for having a look. Still had no time to do more complete analysis if integration of changes in the planning-scene suffers from a similar issue (I think I observed a couple of times that collision objects were not correcty added/removed before the the planning but that is not verified).